deploy-board/deploy_board/static/js/components/capacitycomponents.js

```diff
@@ -214,30 +257,95 @@ Vue.component("static-capacity-config", {
  * In this case, both min size and max size buttons are shown
  */
 Vue.component("asg-capacity-config", {
-    template: '<div class="form-group">\
-    <label for="capacity" class="deployToolTip control-label col-xs-4" title="Number of hosts for this service">\
-        Capacity\
-    </label>\
-            <div class="col-xs-2">\
-                <div class="input-group" >\
-                    <span class="input-group-addon">Min Size</span>\
-                    <input class="form-control" v-bind:value="minsize" v-on:change="updateMinSize($event.target.value)" @keydown.enter.prevent=""></input>\
-                </div>\
-            </div>\
-            <div class="col-xs-2">\
-                <div class="input-group" >\
-                    <span class="input-group-addon">Max Size</span>\
-                    <input class="form-control" v-bind:value="maxsize" v-on:change="updateMaxSize($event.target.value)" @keydown.enter.prevent=""></input>\
-                </div>\
-            </div>\
-    </div>',
-    props: ['minsize', 'maxsize'],
-    methods: {
-        updateMinSize: function (value) {
-            this.$emit('minchange', value)
+    template: `<div>
```

Comment:
is it an extra div? doesn't seem used, maybe consumed somewhere else?

Reply:
No, this groups
```html
 <form-warning v-show="showSizeWarning" :alert-text="sizeWarning"></form-warning>
 <form-error v-show="showSizeError" :alert-text="sizeError"></form-error>
```
Otherwise weird layout shows.